### ISSUE TITLE
How to use this lib on ionic 1 project.

### ISSUE THREAD

**Issue by abdulwahid24** [zxbodya/angular-webpack-seed#8]
Hi,

This might seem a silly question but I couldn't understand how can I use this on existing ionic 1 project.

Please do let me know what would be the process.

Thanks

**Comment by zxbodya** [comment]
Yes, It can be adopted. Eventually one of the first projects where is was used - was old ionic1 project :)

Migration process is relatively simple - you can start using your existing sources as is, by creating  entry module where importing the same files that were included as script tags in html. When building for ionic you will also need to change `publicPath` for output bundle to be relative, and path for output (in make-webpack-config). Also if you do not have proper DI annotations or working ng-annotate - disable scripts minimization, or fix annotations.

Basically, this should be minimal changes, required to get it working.

next steps, I would suggest when updating project:

- replace 'templateUrl', by template and directly requiring html file
- doing `eslint --fix` and reviewing the results (be careful with this one)

**Comment by abdulwahid24** [comment]
Thanks, @zxbodya I would try it as per your instructions and will get back to you.

**Comment by masthisis** [comment]
@abdulwahid24  did you get it to work ???
